From 5083a83aafb01ff278aaa7f14e6e15bfee98e4e3 Mon Sep 17 00:00:00 2001
From: Robert Bogart <robert.bogart@joyent.com>
Date: Mon, 16 Sep 2019 17:54:41 +0000
Subject: [PATCH] MANTA-4552 Rebalancer agent test init automation breaks if
 /var/tmp/rebalancer does not already exist Reviewed by: Rui Loura
 <rui.loura@joyent.com> Approved by: Rui Loura <rui.loura@joyent.com>

---
 test/agent/test_init.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/test/agent/test_init.sh b/test/agent/test_init.sh
index befe1b7..f16abba 100755
--- a/test/agent/test_init.sh
+++ b/test/agent/test_init.sh
@@ -158,7 +158,7 @@ do
     insert_task "$uuid" "$i"
 
     if [[ "$storage_id" -eq "localhost" ]]; then
-        mkdir "$object_dir"
+        mkdir -p "$object_dir"
         cp "$i" "$object_dir"
     else
         curl -X PUT -T $i "http://$storage_id/$account/$(basename $i)"
-- 
2.21.0

